src/cursor/abstract_cursor.ts

```diff
@@ -319,7 +319,7 @@ export abstract class AbstractCursor<
     callback?: Callback<void>
   ): Promise<void> | void {
     if (typeof iterator !== 'function') {
-      throw new MongoDriverError('Missing required parameter `iterator`');
+      throw new MongoInvalidArgumentError('Missing required parameter `iterator`');
```

Comment:
```suggestion
      throw new MongoInvalidArgumentError('Iterator must be a function');
```
This is the same comment along the lines of trying to make errors state what "must be"